core/src/main/java/nl/nn/adapterframework/errormessageformatters/XslErrorMessageFormatter.java

```diff
@@ -57,48 +57,39 @@ public Message format(String errorMessage, Throwable t, INamedObject location, M
 
 		if (StringUtils.isNotEmpty(getStyleSheet()) || StringUtils.isNotEmpty(getXpathExpression())) {
 			try {
-				Transformer errorTransformer;
+				TransformerPool transformerPool;
 
 				if (StringUtils.isNotEmpty(getStyleSheet())) {
-					URL url = ClassUtils.getResourceURL(this, styleSheet);
-					errorTransformer = XmlUtils.createTransformer(url);
-				}
-				else {
-					String xpath = getXpathExpression();
-//					if (StringUtils.isEmpty(xpath)) {
-//						xpath="/errorMessage/@message";
-//					}
-					errorTransformer = XmlUtils.createTransformer(XmlUtils.createXPathEvaluatorSource(xpath));
+					Resource xsltSource = Resource.getResource(this, getStyleSheet());
+					transformerPool = TransformerPool.getInstance(xsltSource, 0);
+				} else {
+					transformerPool = TransformerPool.getXPathTransformerPool(null, getXpathExpression(), null, false, getParameterList());
```

Comment:
I think this code can be further simplified by using TransformerPool.configureTransformer()